apps/web/src/components/layout/components/PrivatePageLayout.tsx

```diff
@@ -62,6 +63,8 @@ export function PrivatePageLayout() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  useOptInRedirect();
```

Comment:
The `PrivatePageLayout` is a top-level component that also has access to the router location, that's why the opt-in hook was placed here.

Reply:
The issue here is that `PrivatePageLayout` is also used in community version, the `useOptInRedirect` calls clerk `useUser()`